pkg/app/piped/eventwatcher/eventwatcher.go

```diff
@@ -237,7 +237,7 @@ func (w *watcher) updateValues(ctx context.Context, repo git.Repo, repoID string
 
 	milestone, ok := w.milestoneMap.Load(repoID)
 	if !ok {
-		milestone = 0
+		milestone = int64(0)
```

Comment:
nit: Since assertion is done later so it could be hard to know why do we need this explicit declaration. So to be more clear:

``` golang
var milestone int64 = 0
if v, ok := w.milestoneMap.Load(repoID); ok {
   milestone = v.(int64)
}
```

Reply:
Right